app/views/questionings/_question.html.erb

```diff
@@ -0,0 +1,59 @@
+          <% @form.questionings.each do |qing| %>
+            <li class="form-question form-item">
+
+            <div class="form-item-content">
+              <%# Standard question? %>
+              <% if qing.standardized? %>
+                <i class="standardized fa fa-certificate"></i>
+              <% else %>
+                <%# Spacing placeholder for styling purposes %>
+                <div class="standardized">
+                </div>
+              <% end %>
+
+              <%# Question type icon %>
+              <div class="type"><%= form_item_icon(qing.qtype_name) %></div>
+
+              <div class="base-info">
+                <%# Question short code %>
+                <span class="code">[<%= qing.code %>]</span>
+
+                <%# Question text %>
+                <span class="name"><%= qing.name %></span>
+
+                <%# Required question? %>
+                <%= content_tag(:span, '*', class: 'required-sym') if qing.required %>
+
+                <%# Tags %>
+                <% if qing.tags.length != 0 %>
+                  <% qing.tags.each do |tag| %>
+                    <span class="token-input-token-elmo"><%= tag.name %></span>
+                  <% end %>
+                <% end %>
+              </div>
+
+            <%# Edit and delete %>
+              <div class="actions">
+
+                <%# Conditional on another question? %>
+                <% if qing.has_condition? %>
+                <div class="condition">
+                  <i class="fa fa-link"></i>
+                  <%# TODO: display question number of linked question %>
+                  <%#= qing.ref_qing %>
+                  <span>#n.n</span>
+                </div>
+                <% end %>
+
+                <%= link_to content_tag(:i, '', class: 'fa fa-pencil'), edit_questioning_path(qing) %>
+
+                <%# TODO: replace with shorter delete code %>
+                <a href="../../questionings/<%= qing.id %>"
```

Comment:
Oh! No what should happen is you setup a JS event handler for the link that calls DELETE /qing-groups/<id>. Then you do e.preventDefault() to stop the link from going anywhere.

We can pair on this if you like. We should probably setup a pairing session for today or tomorrow.